src/packer.pkr.hcl

```
@@ -131,8 +131,15 @@ build {
   }
 
   provisioner "shell" {
-    execute_command = "chmod +x {{ .Path }}; sudo env {{ .Vars }} {{ .Path }} ; rm -f {{ .Path }}"
     script          = "src/post_setup.sh"
+    execute_command = "{{ .Vars }} bash '{{ .Path }}'"
+    skip_clean      = true
+  }
+
+  provisioner "shell" {
+    // We need to call bash here because /tmp has the noexec bit on it
```

Use consistent comment syntax:
```suggestion
    # We need to call bash here because /tmp has the noexec bit on it
```